lib/tasks/convert.rake

```diff
@@ -0,0 +1,27 @@
+namespace :convert do
+  namespace :roles do
+    desc "migrate NCR env var budget approvers to Roles"
+    task ncr_budgets: :environment do
+      User.with_email_role_slug!(
+        ENV['NCR_BA61_TIER1_BUDGET_MAILBOX'] || 'communicart.budget.approver@gmail.com',
+        'BA61_tier1_budget_approver',
+        'ncr'
+      )
+      User.with_email_role_slug!(
+        ENV['NCR_BA61_TIER2_BUDGET_MAILBOX'] || 'communicart.ofm.approver@gmail.com',
+        'BA61_tier2_budget_approver',
+        'ncr'
+      )
+      User.with_email_role_slug!(
+        ENV['NCR_BA80_BUDGET_MAILBOX'] || 'communicart.budget.approver@gmail.com',
+        'BA80_budget_approver',
+        'ncr'
+      )
+      User.with_email_role_slug!(
+        ENV['NCR_OOL_BA80_BUDGET_MAILBOX'] || 'communicart.budget.approver@gmail.com',
+        'OOL_BA80_budget_approver',
+        'ncr'
+      )
+    end
+  end
+end
```

Comment:
If this is required locally and we _don't_ keep it in `seeds` or a migration, we should run the rake command when [bootstrapping](https://github.com/18F/C2/blob/master/script/bootstrap).